runtime/trappist/src/lib.rs

```diff
@@ -700,7 +755,8 @@ construct_runtime!(
 		Uniques: pallet_uniques = 43,
 		Scheduler: pallet_scheduler = 44,
 		Preimage: pallet_preimage = 45,
-		LockdownMode: pallet_lockdown_mode = 46,
+		SafeMode: pallet_safe_mode = 46,
```

Comment:
I'm not sure about re-using the same index here. To be on the safe side, I would skip index 46 (and leave a comment). It shouldn't be a problem though because we will increase the runtime `transaction_version` for the next runtime upgrade of Trappist on Rococo. But I will still prefer to err on the safe side on this one.

Reply:
You're right, better be on the safe side. Changed in [33b1736](https://github.com/paritytech/trappist/pull/305/commits/33b173617174bd419a5246670ac3699375c25d51).